cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -219,7 +221,11 @@ private CristinBookReport extractCristinBookReport() {
     }
 
     private String extractNumberOfPages() {
-        return extractCristinBookReport().getNumberOfPages();
+        String numberOfPages = extractCristinBookReport().getNumberOfPages();
+        if (numberOfPages == null) {
```

Comment:
StringUtils.isBlank(numberOfPages). Also in general the style is to use the  isNull(), notNull()  methods